Terasology/Health
# feat: Add a red flash on receiving damage

### What this PR does

Adds a red flash on the player receiving damage (the death screen background). Overlay stays for 150ms.

![image](https://user-images.githubusercontent.com/17286005/100011145-08b5e600-2e1d-11eb-8204-d0bac1258ce5.png)

### How to Test

Start a game with WildAnimals enabled and spawn in a hostileDeer. You should see the flash when it attacks you.

Requires https://github.com/MovingBlocks/TeraNUI/pull/42
